### PR TITLE
Rf7ECYx3-CRUD operation for route

### PR DESCRIPTION
**Pull request by Julesbi**
# Crud operation for routes

## Overview
In this feature, I have completed CRUD operation for routes where an operator can be able

 - to create
 - update
  - delete a route, and
  - users are able to view the route's details 
## Testing 

To test this follow these steps 

1. Pull this branch
2. run `npm install` to install all dependencies in `package.json`
3. run `touch .env` to create .env file on the root folder
4. Based on database configuration enter your DB_URL
5. run `npm run migrate` to create the database
6. run `npm build` to build the app with babel
7.  run `npm run dev` to start a server 
8. use postman to test API

## Relevant Pivotal tracker
[CRUD operation for routes](https://trello.com/c/Rf7ECYx3/7-crud-operations-for-routes)

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -1,29 +1,56 @@
+<<<<<<< HEAD
```
Comment (niyontwali) [inline .github/workflows/main.yml]: Please remove `<<<<<<< HEAD` in your codes on line 1

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -1,29 +1,56 @@
...
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
 # This is a basic workflow to help you get started with Actions
 
 name: CI
+=======
```
Comment (niyontwali) [inline .github/workflows/main.yml]: remove also this on line 12

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -1,29 +1,56 @@
...
+      matrix:
+        node: ['12', '14', '16']
+    name: Node ${{ matrix.node }} sample
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
+      - name: Setup node
+        uses: actions/setup-node@v2
         with:
+<<<<<<< HEAD
```
Comment (niyontwali) [inline .github/workflows/main.yml]: This line also need to be removed

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -1,29 +1,56 @@
...
+          node-version: ${{ matrix.node }}
+=======
```
Comment (niyontwali) [inline .github/workflows/main.yml]: This line also need to be removed

File: .github/workflows/main.yml [hunk 5]
```diff
@@ -1,29 +1,56 @@
...
           node-version: '14'
+=======
```
Comment (niyontwali) [inline .github/workflows/main.yml]: This line also need to be removed

**Review by niyontwali — CHANGES_REQUESTED** [review]
Thanks for your PR, you really did a great work. Please address the inserted comments.

File: .github/workflows/main.yml (hunk 4 above)
Comment (Julesbi) [reply .github/workflows/main.yml]: Thanks were removed

File: .github/workflows/main.yml (hunk 3 above)
Comment (Julesbi) [reply .github/workflows/main.yml]: Thanks were removed

File: .github/workflows/main.yml (hunk 2 above)
Comment (Julesbi) [reply .github/workflows/main.yml]: Thanks were removed

File: .github/workflows/main.yml (hunk 5 above)
Comment (Julesbi) [reply .github/workflows/main.yml]: Thanks were removed

File: .github/workflows/main.yml (hunk 1 above)
Comment (Julesbi) [reply .github/workflows/main.yml]: Thanks were removed

**Comment by SimonImanigirimpuhwe** [comment]
@niyontwali  Please check if the changes you requested was responded to

**Review by niyontwali — APPROVED** [review]
Thanks, the requested changes were addressed.

**Review by JPaul23 — APPROVED** [review]

**Review by niyontwali — APPROVED** [review]

**Pull request merged**